### PR TITLE
Include OTLP Events documentation

### PR DESCRIPTION
**Pull request by jsubirat**
## Give us some context

* What problems does this PR solve?
Adds documentation on how to use the OpenTelemetry Events API to send logs to New Relic.

* Add any context that will help us review your changes such as testing notes,
  links to related docs, screenshots, etc.
Links to related docs were included in the contributed test.

* If your issue relates to an existing GitHub issue, please link to it.
None.

### CODE REVIEW

**Comment by jsubirat** [comment]
Moved the PR to draft while we review the documentation from our side.

File: src/content/docs/mlops/bring-your-own/getting-started-byo.mdx [hunk 1]
```diff
@@ -85,7 +85,7 @@ Done! To check your application go to **[one.newrelic.com](https://one.newrelic.
 
 ## Examples [#examples]
 We created these notebooks in Google colab so you can try them out:
-  1. [Try out](https://github.com/newrelic-experimental/ml-performance-monitoring/blob/main/examples/XGBoost_on_Boston_housing_prices_dataset.ipynb) an XGBoost model on [Boston housing prices](https://scikit-learn.org/stable/modules/generated/sklearn.datasets.load_boston.html) dataset. Open in [colab](https://colab.research.google.com/github/newrelic-experimental/ml-performance-monitoring/blob/main/examples/XGBoost_on_Boston_housing_prices_dataset.ipynb).
+  1. [Try out](https://github.com/newrelic-experimental/ml-performance-monitoring/blob/main/examples/XGBoost_on_Boston_housing_prices_dataset.ipynb) an XGBoost model on [Boston housing prices](https://scikit-learn.org/0.15/modules/generated/sklearn.datasets.load_boston.html) dataset. Open in [colab](https://colab.research.google.com/github/newrelic-experimental/ml-performance-monitoring/blob/main/examples/XGBoost_on_Boston_housing_prices_dataset.ipynb).
```
Comment (jsubirat) [inline src/content/docs/mlops/bring-your-own/getting-started-byo.mdx:88]: Not sure why this change was introduced, I didn't do it from my side 🤷‍♂️

Comment (rhetoric101) [reply src/content/docs/mlops/bring-your-own/getting-started-byo.mdx:88]: @jsubirat I will roll this change since the MLOps team is going to create a separate Jira to correct this.

**Comment by rhetoric101** [comment]
I am closing temporarily to trigger a new build when I reopen it.

File: src/content/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/view-your-data/opentelemetry-events.mdx [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+  - Open source telemetry integrations
+  - OpenTelemetry
+metaDescription: You can query experimental OpenTelemetry events in New Relic.
+---
+
+import otelEventsQueryScreenshot from 'images/otel-events.png'
+
+
+<Callout title="Limited preview">
+  If you sign up for the limited preview of the Events API, you can query OpenTelemetry events in New Relic. Let us know in our [CNCF Slack channel, #otel-newrelic](https://cloud-native.slack.com/archives/C024DRQ63UP) if you'd like to participate.
```
Comment (zuluecho9) [inline src/content/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/view-your-data/opentelemetry-events.mdx:14]: Is this an OpenTelemetry feature preview and not our preview? In that case I'd make that more clear either way. if it's our preview, we should just say 'preview' and not 'limited preview'. If this is first place we introduce this, we should probably explain 'This feature is in preview.' or whatever.

File: src/content/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/view-your-data/opentelemetry-events.mdx [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+</Callout>
+
+OpenTelemetry recently introduced the experimental [Events API](https://opentelemetry.io/docs/reference/specification/logs/event-api/) in its specification. This API allows you to create an `EventLogger`, which can emit `Events` as `LogRecords` containing two special attributes:
+* `event.domain`: The domain of the emitted events. It needs to be be set during the creation of the `EventLogger`.
```
Comment (zuluecho9) [inline src/content/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/view-your-data/opentelemetry-events.mdx:18]:
```suggestion
* `event.domain`: The domain of the emitted events. It needs to be set during the creation of the `EventLogger`.
```

File: src/content/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/view-your-data/opentelemetry-events.mdx [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+* `event.name`: The event name. This attribute is what New Relic uses to map an OpenTelemetry event into a [New Relic eventType](https://docs.newrelic.com/docs/data-apis/custom-data/custom-events/data-requirements-limits-custom-event-data/#general)
```
Comment (zuluecho9) [inline src/content/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/view-your-data/opentelemetry-events.mdx:19]:
```suggestion
* `event.name`: The event name. This attribute is what New Relic uses to map an OpenTelemetry event to a [New Relic eventType](/docs/data-apis/custom-data/custom-events/data-requirements-limits-custom-event-data/#general)
```

Comment (zuluecho9) [inline src/content/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/view-your-data/opentelemetry-events.mdx:19]: This isn't clear to me. what does it mean to map an OTel event to a `eventType'? `eventType` is an attribute on custom events I think, so this isn't that clear. is this saying that this is creating a custom event and setting the eventType to whatever event.name is? We should maybe make that a bit clearer, maybe saying something like 'This feature brings in events via our Event API and brings them in as custom events with the name set with the eventType attribute.' Or something like that.

Comment (zuluecho9) [inline src/content/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/view-your-data/opentelemetry-events.mdx:19]: I think we can make this clearer, saying something like 'We bring in these events via our Event API as a custom event, using the `eventType` attribute to set the event name.' Something like that.

File: src/content/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/view-your-data/opentelemetry-events.mdx [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+
```
Comment (zuluecho9) [inline src/content/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/view-your-data/opentelemetry-events.mdx:20]: i don't think we should say 'eventType' unless we're specifically referencing the attribute. Once you set the attribute, that just becomes it's 'event type' and so we should use that like that, more informally. So like: You'll be able to find these events in New Relic as the `ApiRequest` event type.

**Review by zuluecho9 — APPROVED** [review]
